Lib/test/test_linecache.py

```diff
@@ -281,6 +281,19 @@ def test_loader(self):
         self.assertEqual(linecache.getlines(filename, module_globals),
                          ['source for x.y.z\n'])
 
+    def test_frozen(self):
+        filename = '<frozen fakemodule>'
+        module_globals = {'__file__': FILENAME}
+        empty = linecache.getlines(filename)
+        self.assertEqual(empty, [])
+        lines = linecache.getlines(filename, module_globals)
+        self.assertGreater(len(lines), 0)
+        lines_cached = linecache.getlines(filename)
+        self.assertEqual(lines, lines_cached)
```

Comment:
```suggestion
        self.assertListEqual(lines, lines_cached)
```

the diff in case of failure will be nicer

Reply:
I applied the others. I'm not 100% sure on this. There can't be a "small difference" between these. It's either the same, or completely different. I think `Equal` here has a slightly better semantic (that they should be the same because it's a cached data). `List` introduces a little bit noise.

Reply:
I'm fine with this argument!